test_all.py

```diff
@@ -893,10 +893,10 @@ def test_unit_actions():
     ucontrol targetp @unit 1 0 0 0
     ucontrol target 9 10 1 0 0
     ucontrol target 0 0 0 0 0
-    ucontrol itemTake from item amount 0 0
-    ucontrol itemTake from item amount 0 0
-    ucontrol itemDrop to amount 0 0 0
-    ucontrol itemDrop to amount 0 0 0
+    ucontrol itemTake "container1" "copper" 1 0 0
```

Comment:
Does this work in-game? Or would it need to be

```
ucontrol itemTake container1 @copper 1 0
```

?

Reply:
You're absolutely right. I was asleep at the wheel when fixing that test. It's fixed now.